core/App/components/modals/CommonDeleteModal.tsx

```diff
@@ -82,48 +74,53 @@ const CommonDeleteModal: React.FC<CommonDeleteModalProps> = ({ visible, onSubmit
   return (
     <Modal visible={visible} animationType="slide">
       <FauxNavigationBar title={t('CredentialDetails.RemoveFromWallet')} />
-      <SafeAreaView style={[styles.container]}>
-        <View style={[styles.main]}>
-          <View style={[{ marginBottom: 25 }]}>
-            <Text style={[TextTheme.title]}>{t('CredentialDetails.RemoveTitle')}</Text>
-          </View>
-          <View>
-            <Text style={[TextTheme.normal]}>{t('CredentialDetails.RemoveCaption')}</Text>
-          </View>
-          <View style={{ marginTop: 25 }}>
-            <Dropdown
-              title={t('CredentialDetails.YouWillNotLose')}
-              content={[t('CredentialDetails.YouWillNotLoseListItem1'), t('CredentialDetails.YouWillNotLoseListItem2')]}
-            />
-          </View>
-          <View style={{ marginTop: 25 }}>
-            <Dropdown
-              title={t('CredentialDetails.HowToGetThisCredentialBack')}
-              content={[t('CredentialDetails.HowToGetThisCredentialBackListItem1')]}
-            />
-          </View>
-          <View style={[styles.actions]}>
-            <View style={[{ marginBottom: 5 }]}>
-              <Button
-                title={t('CredentialDetails.RemoveFromWallet')}
-                accessibilityLabel={t('CredentialDetails.RemoveFromWallet')}
-                testID={testIdWithKey('ConfirmRemoveButton')}
-                onPress={() => onSubmit && onSubmit()}
-                buttonType={ButtonType.Primary}
+      <ScrollView>
```

Comment:
Should scroll view be wrapped inside the safe area context?

Reply:
I tried that but it cuts off the cancel button